src/library_gl.js

```diff
@@ -1257,7 +1257,12 @@ var LibraryGL = {
 #if GL_ASSERTIONS
     GL.validateGLObjectID(GL.textures, texture, 'glBindTexture', 'texture');
 #endif
-    GLctx.bindTexture(target, texture ? GL.textures[texture] : null);
+    if( texture >= 0 )
```

Comment:
indentation and formatting should match the rest of the code, that is, `if (texture >= 0) {` on the first line etc.